app/assets/stylesheets/hackney_property.scss

```diff
@@ -11,3 +11,19 @@
 .hackney-property-tenure-orange {
   background: hackney-colour("warning");
 }
+
+.hackney-cautionary-contact {
+  color: white !important;
```

Comment:
does it break if you move this rule to `.hackney-cautionary-contact-table`?

Reply:
yes

Reply:
that govuk css is very bad =o(